src/views/Room.vue

```diff
@@ -80,6 +83,7 @@
 <script>
 import { API } from "@/common-http.js";
 import VideoQueueItem from "@/components/VideoQueueItem.vue";
+import convertSecondstoHHMMSS from "../timestamp.js";
```

Comment:
Change to `@/timestamp.js`

Reply:
`@` refers to the base directory, so if we move this file to a different directory in the future, we don't have to modify our imports.